ethereum_history_api/oracles/src/noir/oracles/common/const.ts

```diff
@@ -1,9 +1,13 @@
 export const MODULUS = 21888242871839275222246405745257275088696311157297823662689037894645226208583n;
+export const BITS = 2n;
+export const BITS_IN_BYTE = 8n;
 export const U1_LEN = 1;
 export const BYTES32_LEN = 32;
 export const ADDRESS_LEN = 20;
 export const U64_LEN = 8;
-export const BITS_IN_BYTE = 8n;
+export const U64_MAX_VALUE = BITS ** (BigInt(U64_LEN) * BITS_IN_BYTE) - 1n;
```

Comment:
Also, I think it would be much more readable if we were able to write U64_MAX = 2 ** 64. Let's maybe remove no magic number rule in const.ts? What do you think @LogvinovLeon?